pkg/tuned/dialog.js

```diff
@@ -275,18 +280,13 @@
         button.on('click', open_dialog);
 
         tuned_service.addEventListener("changed", update_button);
-        update_button();
```

Comment:
I think this is okay. We always get an initial `change` event when the service object initializes.

Reply:
On second thought, please put it back. This is a theoretical race if the service object can finish reading the properties before we attach the `changed` handler above.